### PR TITLE
crystal-lang 0.20.0

### PR DESCRIPTION
**Pull request by asterite**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Thank you in advance! 🙇

### CODE REVIEW

**Comment by ilovezfs** [comment]
hurray!

File: Formula/crystal-lang.rb [hunk 1]
```diff
@@ -36,14 +23,14 @@ class CrystalLang < Formula
...
-    sha256 "2c9aebfefe2aca46eeda1e5a3fd6a91e3177af8f324ea23ebf8b5cad3c87ad2d"
+    url "https://github.com/crystal-lang/crystal/releases/download/0.19.4/crystal-0.19.4-1-darwin-x86_64.tar.gz"
+    version "0.19.4"
+    sha256 "ec9f21f29035b76b2bc32bd3d975aa057f68007191cc6aab1535effe4c1c60d5"
   end
 
   resource "shards" do
-    url "https://github.com/ysbaddaden/shards/archive/v0.6.4.tar.gz"
-    sha256 "5972f1b40bb3253319f564dee513229f82b0dcb8eea1502ae7dc483a9c6da5a0"
+    url "https://github.com/ysbaddaden/shards/archive/v0.7.0.tar.gz"
```
Comment (asterite) [inline Formula/crystal-lang.rb]: Ah, good. I'll send a PR once this is merged, if that's OK with you

Comment (ilovezfs) [inline Formula/crystal-lang.rb]: @asterite I took care of it while pulling.

**Comment by ilovezfs** [comment]
@asterite Thanks for doing this PR and for the LLVM 3.9 compatibility without patching :tada:

File: Formula/crystal-lang.rb [hunk 2]
```diff
@@ -1,24 +1,11 @@
...
-    # changes already merged upstream to fix compilation with LLVM 3.9
-    # https://github.com/crystal-lang/crystal/pull/3439
-    patch do
-      url "https://github.com/crystal-lang/crystal/commit/13b11d7.patch"
-      sha256 "d981515791c48ae7fce0e906b0eec934fd622987a87f0614b3c91c71b0966b66"
-    end
-  end
-
   bottle do
-    rebuild 1
```
Comment (ilovezfs) [inline Formula/crystal-lang.rb]: @asterite in the future please don't edit the bottle block manually

Comment (asterite) [inline Formula/crystal-lang.rb]: Sure! I didn't know what I had to do there so I reverted it to what was there last time I touched this file. Thank you again for promptly reviewing and merging this 🎉

**Pull request closed without merging**